### ISSUE TITLE
Malformed `TRAINABLE_ATTRIBUTE_NOTE` in batch normalization documentation

### ISSUE THREAD

**Issue by joe-antognini** [tensorflow/tensorflow#35648]
## URL(s) with the issue:

https://www.tensorflow.org/api_docs/python/tf/keras/layers/BatchNormalization?version=stable

## Description of issue (what needs changing):

After the references list there is a stray malformed tag:

{ {TRAINABLE_ATTRIBUTE_NOTE}}

I suspect that this is supposed to resolve to the note that `moving_mean` and `moving_variance` are placed in `UPDATE_OPS` and need to be executed alongside the training op.  (This note is present in the `tf.layers` doc: https://www.tensorflow.org/versions/r1.15/api_docs/python/tf/layers/batch_normalization)  But without knowing what this tag refers to I can't really say for sure.  (The tag is present in only three places in the Tensorflow codebase and shows up malformed on the website in each case.)

**Comment by joe-antognini** [comment]
Recent updates to the Tensorflow website seem to have resolved this issue on the link posted, but the issue continues to exist for the v1.15 docs: https://www.tensorflow.org/versions/r1.15/api_docs/python/tf/keras/layers/BatchNormalization.

**Comment by rmothukuru** [comment]
@joe-antognini,
As **`Tensorflow 1.x`** is not actively supported and as we recommend `Developer Community` to **`upgrade`** to **`Tensorflow 2.x`**, can you please let us know if we can close this issue, as it has been resolved in the **`actively supported Version (2.x)`**? Thanks!

**Comment by joe-antognini** [comment]
Sure, we can close this.
